css-layout-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-two-thirds {
+  align-items: center;
```

Comment:
Don't pollute your layout classes. `column-` classes should only include `flex-basis`